services/transactions/transactions_test.go

```diff
@@ -0,0 +1,199 @@
+package transactions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
```

Comment:
I see that you started using testify in other PRs - in general we don't use testing frameworks. I think go-cmp should be enough for our needs.

I don't want to block this PR, and there's already code in here using testify, but I do think we should be consistent about which testing libraries we (as a company) use.

Reply:
I think a good first step would be to remove the use of the `assert` package. I haven't worked enough with mocking in Go to know if testify/mock is really needed.

Reply:
Yes I introduced `testify/mock` as it provides a simple way to mock calls to other functions and control their results. 
I'm not familiar with `golang/mock`, I'll give it a try!

Reply:
It's done for transactions, I'll remove `testify` from accounts in another PR